### ISSUE TITLE
Shift+Space to scroll up in programs

### ISSUE THREAD

**Issue by rightaway** [zeit/hyper#2602]
In scrollable programs like `man` and `less`, pressing `Space` scrolls one page down. Pressing `Shift+Space` to scroll up would be very helpful and is a convention in many programs but doesn't work in hyper, could this feature be added?

**Comment by albinekb** [comment]
I like the idea! @chabou isn't this possible already using our keymaps? Is there an event to trigger scroll up, if not, is it possible to add it? 🤔

**Comment by chabou** [comment]
I didn't know this behaviour.
Can you give some examples of these programs?

`space` in less is a less-specific feature. And there is already some shortcuts : `ctrl+f` for page down like space and `ctrl+b` for page up.

**Comment by rightaway** [comment]
Many programs like for viewing pdfs, images, and pretty much all web browsers.

**Comment by chabou** [comment]
I see, GUI programs.
In command line, `space` and `shift+space` are not relevant, imo. 

But it could be handy to have shortcut to scroll up/down like mouse wheel.
In iTerm : 
`command+up/down`: scroll 1 line up/down
`command+pageup/pagedown` or `shift+pageup/pagedown`: scroll 1 page up/down

**Comment by rightaway** [comment]
I've used terminals a while ago (don't remember which now) where shift+space worked for doing the opposite of space in programs like `man` and `less` and it was a really nice usability point.

**Comment by chabou** [comment]
Related ressource: https://unix.stackexchange.com/questions/88579/shift-space-in-less

**Comment by albinekb** [comment]
@chabou Not saying we should add this as default. But we should def support it in the config/custom keymaps so people that want to have it can configure it like that 👍

**Comment by chabou** [comment]
@albinekb agreed but adding keymap for scrolling (good thing!) will not work in `less`.

**Comment by jcrben** [comment]
It would be good for Shift+Space to be recognized. According to https://emacs.stackexchange.com/a/13957 standards are emerging:
> Lately, the situation is changing, because there have been two proposals to standardize on escape sequences. One is LeoNerd's libtermkey with the syntax ESC [ codepoint ; modifier u. Another is Thomas Dickey's xterm with the syntax ESC [ 2 7 ; modifier ; codepoint ~. Current versions of xterm (≥216) can be configured for either syntax by setting the formatOtherKeys resource; the feature must be activated by setting the modifyOtherKeys to a nonzero value.

this would let me accomplish https://stackoverflow.com/questions/279959/how-can-i-make-shiftspacebar-page-up-in-vim without the intermediate hack...
